reference_frame: "map"
wheelRadius: 0.08
wheelSeperator: 0.48
velRedWheels: 0.9 # 0.1
velRedArm: 0.20 # 0.25
mpc:
  timeHorizon: 15
  motionTypes:
    - "/navigation"
    - "/navigation"
  goals:
    - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, -1.0, 0.0, 2.0, 0.0]
    - [0.0, 2.0, 0.0, 0.0, 0.0, 0.0, -1.0, 0.0, 2.0, 0.0]
  timeStep: 1.0
  maxError: 0.1
  rate: 1.0
  navigation:
    safetyMarginBase: 0.35
    safetyMarginArm: 0.15
    #arm, contour, lag, orientation, slack, u, q_dot
    weights: [0.7, 3.5, 3.5, 1.0, 1000.0, 0.1, 10.0]
    #pos, orientation, arm
    errorWeights: [1.0, 1.0, 1.0]
  armMotion:
    safetyMarginBase: 0.35
    safetyMarginArm: 0.15
    weights: [0.7, 0.0, 0.0, 0.0, 1000000000.0, 10000000.0, 10.0]
    errorWeights: [0.0, 0.0, 1.0]
  baseMotion:
    safetyMarginBase: 0.35
    safetyMarginArm: 0.15
    weights: [0.00001, 3.5, 1.5, 1.0, 1000000000.0, 0.1, 100000.0]
    errorWeights: [1.0, 1.0, 0.0]
